content/en/platform/corda/4.11/community/apps/bankinabox/user-interface/admin-ui-guide.md

```diff
@@ -267,7 +267,7 @@ To find a specific recurring payment, use the search bar. After typing three cha
 
 If you wish to see all relevant information for a specific recurring payment, click on the recurring payment and you will be directed to the [Recurring payments view](#recurring-payments-view).
```

Comment:
Add relref here?

Reply:
I skipped the ones which are a link to an anchor in the same files, as turning it into a relref doesn't do any verification - an invalid one will still work. So didn't give any value to do it.